streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordContextStub.java

```diff
@@ -1,81 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This is removed as its functionality is completely replaced with `ProcessorRecordContext` now.